<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="snap_cam" type="snap_cam_publisher_node" name="snap_cam_optflow_publisher" output="screen">

    <!--
    "optflow" or "highres" -->
    <param name="camera_type" value="optflow"/>

    <!--
    optical flow: QVGA : (320,240)
                  VGA  : (640,480) -->
    <param name="camera_resolution" value="VGA"/>

    <!--
    optical flow: 0: 15, 1: 24, 2: 30, 3: 60, 4: 90, 5: 30, 6: 30 -->
    <param name="camera_fps_idx" value="0"/>

    <!--
    optical flow: integer between 0 and 511 -->
    <param name="exposure" value="100"/>

    <!--
    optical flow: integer between 0 and 255 -->
    <param name="gain" value="50"/>

    <param name="topic_name" value="image"/>
  </node>
</launch>
